hello-world/Cargo.toml

```diff
@@ -7,3 +7,7 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+
+[dev-dependencies]
```

Comment:
I guess there is a way to do this by using only the standard library. Let's try that first. If that doesn't work or is complicated then we can try to use these crates.